content/influxdb3/core/query-data/execute-queries/influxdb-v3-api.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Use the InfluxDB v3 HTTP query API to query data
+seotitle: Use SQL or InfluxQL and InfluxDB v3 HTTP query API
+list_title: Use the v3 query API
+description: >
+  Use SQL or InfluxQL and the InfluxDB v3 HTTP query API to query data in {{< product-name >}}.
+weight: 301
+menu:
+  influxdb3_core:
+    parent: Execute queries
+    name: Use the v3 query API
+influxdb3/core/tags: [query, influxql, sql, python]
+metadata: [InfluxQL, SQL]
+related:
+  - /influxdb3/core/api-compatibility/v1/
+aliases:
+  - /influxdb3/core/query-data/influxql/execute-queries/influxdb-v1-api/
```

Comment:
I'm not sure we want to alias this doc here. There are going to be customers who will likely want to continue to use the v1 query API so they don't have to update their applications or clients to use the new v3 query API.

```suggestion
```